src/main/java/edu/oregonstate/cs361/battleship/BattleshipModel.java

```diff
@@ -61,5 +61,10 @@
         this.computer_cruiser = new Ship("Computer_Cruiser", 3);
         this.computer_destroyer = new Ship("Computer_Destroyer", 2);
         this.computer_submarine = new Ship("Computer_Submarine", 2);
+
+        this.playerHits = new int[0];
+        this.playerMisses = new int[0];
```

Comment:
Also, yes, I think the parseerror is for another user story - my ships get placed automatically, but the user trying to place ships gives a pareserror. So whoever is working on the user placement story will take care of that, I think.